api/mock_history_data.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
This should return data for each of the four browsers, so I'm assuming this structure is sent over for each browser?